corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -90,7 +90,7 @@ def setUp(self):
             self.restore_options,
             default_case_type=PARENT_TYPE,
         )
-        self.sync_log = self.device.last_sync.log
+        self.sync_log = self.device.sync(overwrite_cache=True, version=V1).log
```

Comment:
This is what it did [before](https://github.com/dimagi/commcare-hq/blob/aa13efd674f130c22b0f0e7ac769832684c524f5/corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py#L80-L83). `RestoreConfig` uses [`version=V1` by default](https://github.com/dimagi/commcare-hq/blob/5bd6c3ba709f380d9f14f6643e73ccb044fe8bef/corehq/ex-submodules/casexml/apps/phone/restore.py#L410).